### PR TITLE
Hong branch

### PR DESCRIPTION
**Pull request by hongdong1220**
Updated the README

### CODE REVIEW

File: README.md
Comment (josephdprince) [inline README.md:10]: Brad is not a contributor on this project

File: README.md
Comment (josephdprince) [inline README.md]: "and will be discussed more in the paper". If you are going to say this then you should link the report.

**Comment by josephdprince** [comment]
2 small changes needed before approval

File: README.md
Comment (hongdong1220) [reply README.md:10]: We have him on the paper, so I am including him

File: README.md
Comment (hongdong1220) [reply README.md]: Added the report link

File: README.md
Comment (josephdprince) [inline README.md]: link is broken, I think you need a forward slash instead of a backslash

**Review by josephdprince — APPROVED** [review]
nice

**Pull request merged**
